docs/specification/README.md

```diff
@@ -279,7 +279,7 @@ A metadata record contains arbitrary user data in key-value pairs.
 
 | Bytes | Name | Type | Description |
 | --- | --- | --- | --- |
-| 4 + N | name | String | Example: `map_metadata`. |
+| 4 + N | name | String | Example: `mycompany`. |
```

Comment:
`mycompany` on its own might be a bit oblique, better to make the example specific to a category of metadata.

```suggestion
| 4 + N | name | String | Example: `my_company_name_hardware_info`. |
| 4 + N | metadata | `Map<string, string>` | Example keys: `part_id`, `serial`, `board_revision` |
```